analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -93,7 +93,8 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
         private const val MAX_DISK_CACHE_ENTRY_AGE_SECONDS = 6 * 60 * 60
 
         // See http://maven.apache.org/pom.html#SCM.
-        val SCM_REGEX = Pattern.compile("scm:(?<type>[^:]+):(?<url>.+)")!!
+        val SCM_REGEX = Pattern.compile("scm:(?<type>[^:@]+):(?<url>.+)")!!
```

Comment:
Just wondering, do you think we can / should simplify `[^:@]+` to `\w+`, or do you think we need to support non-word characters as part of the provider? Unfortunately the [Maven docs](http://maven.apache.org/pom.html#SCM) say nothing about which characters are valid for the provider.

Reply:
I think `\w` misses at least "-" and probably some other characters which would be valid.

Reply:
You're right, while `_` is part of `\w`, `-` is not, and we need it e.g. for "git-repo".